src/Dofs/DofHandler.jl

```diff
@@ -322,6 +328,29 @@ function add!(dh::DofHandler, name::Symbol, ip::Interpolation)
     return dh
 end
 
+"""
+    add_global_dofs!(dh::DofHandler, name::Symbol, n::Int)
+
+Add `n` global degrees of freedom to `dh`. The degree of freedom numbers can be obtained
+after closing `dh` with [`get_global_dofs`](@ref).
+"""
+function add_global_dofs!(dh::DofHandler, name::Symbol, n::Int)
+    @assert !isclosed(dh)
+    haskey(dh.global_dofs, name) && throw(ArgumentError(string(":", name, " is already a global dof")))
+    dh.global_dofs[name] = (-n):(-1) # Range shifted in close
+end
+
+"""
+    get_global_dofs(dh::DofHandler, name::Symbol)
+
+Get a `UnitRange{Int}` describing the global dofs associated with `name`.
+Such dofs can be added with [`add_global_dofs!`](@ref).
+"""
+function get_global_dofs(dh::DofHandler, name::Symbol)
```

Comment:
Perhaps
```suggestion
function global_dof_range(dh::DofHandler, name::Symbol)
```
(https://github.com/Ferrite-FEM/Ferrite.jl/blob/17c9ef20a77ebd6315c61ce9cc561c1e8504ef54/src/Dofs/DofTools.jl#L86)

Reply:
Thought about it, but not sure how this plays with `dof_range` which gives you the local dof range in the cell. Global in that respect here refers both to that the dofs are global in nature, and that it is the global (and not local) numbering that is included in the range. 

But if you think that is ok, then I think your naming suggestion is better.

Reply:
I think it has the same meaning: "What range in the global dof numbering is field x" is what both of them correspond to, no?

Reply:
Related to this: How will this interact with the renumbering logic?